src/Modal/ModalPropsTypes.ts

```diff
@@ -0,0 +1,30 @@
+export interface ErrorMessageProps {
```

Comment:
file name should start from small case if it is not component
so modalPropTypes.ts or just types.ts